src/dicg.jl

```diff
@@ -491,52 +515,86 @@ function lazy_dicg_step(
     phi,
     epsilon,
     d;
-    use_extra_vertex_storage=false,
-    extra_vertex_storage=nothing,
-    lazy_tolerance=2.0,
-    memory_mode::MemoryEmphasis=InplaceEmphasis(),
+    variant = "standard",
+    strong_lazification = false,
+    use_extra_vertex_storage = false,
```

Comment:
If this is a specific function for the lazy version of dicg, we always use the extra vertex storage, no? In which case, the default value for `use_extra_vertex_storage` should be `true`.